mesh/mesquite.cpp

```diff
@@ -715,7 +715,7 @@ void MesquiteMesh::elements_get_topologies(const ElementHandle
 
 MesquiteMesh::~MesquiteMesh()
 {
-   MsqPrintError err(cerr);
+   MsqPrintError err(mfem::err.GetStream());
```

Comment:
Oops, need to update this: `mfem::err.GetStream() -> mfem::err`.

Reply:
Done in Veselin's update.